src/vs/workbench/contrib/terminal/browser/terminalGroupService.ts

```diff
@@ -106,6 +120,55 @@ export class TerminalGroupService extends Disposable implements ITerminalGroupSe
 		return group;
 	}
 
+	async showPanel(focus?: boolean): Promise<void> {
```

Comment:
Can we remove showPanel from ITerminalService?

Reply:
it's referenced in a ton of places, but yes. would just require adding `ITerminalGroupService `to the places that call that

Reply:
K, let's not do that here then 👌 should be a relatively easy debt item https://github.com/microsoft/vscode/issues/127111